app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -215,6 +215,13 @@
                 {{input type=field.type value=(mut (get data.speaker field.fieldIdentifier))
                         id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))}}
               {{/if}}
+            {{else if (eq field.fieldIdentifier 'sponsorshipRequired')}}
+              <div class="field">
+                {{ui-radio label=(t 'Yes') current=data.speaker.sponsorshipRequired name='speaker_sponsorshipRequired_required' value='yes' onChange=(action (mut data.speaker.sponsorshipRequired))}}
```

Comment:
This is duplication. Make a dictionary-like done for genders and use
{{#each}} with a single {{ui-radio}}.